rfc9204.md

```diff
@@ -657,14 +640,14 @@ HPACK defines string literals to begin on a byte boundary.  They begin with a
 single bit flag, denoted as 'H' in this document (indicating whether the string
 is Huffman-coded), followed by the Length encoded as a 7-bit prefix integer, and
 finally Length bytes of data. When Huffman encoding is enabled, the Huffman
-table from {{Section B of RFC7541}} is used without modification and Length
+table from {{Section B of RFC7541}} is used without modification, and Length
```

Comment:
```suggestion
table from {{Section B of RFC7541}} is used without modification and Length
```
Since the Huffman table isn't a parenthetical phrase, I'm not sure adding this comma makes sense.